src/index.html

```diff
@@ -11,7 +11,104 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" >
+
+      <fieldset class="form__general">
+        <legend>Personal information</legend>
+        <label class="form__field">Surname: <input type="text"
+          name="surname"
+          autocomplete="off">
```

Comment:
```suggestion
        <label class="form__field">
          Surname:
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
```
follow the [code style guide](https://mate-academy.github.io/style-guides/htmlcss.html) 😉